view/vw_wastewater_node.sql

```diff
@@ -84,9 +89,9 @@ UPDATE qgep.od_wastewater_networkelement
   SET
        identifier = NEW.identifier
      , remark = NEW.remark
-     , dataowner = NEW.dataowner
-     , provider = NEW.provider
-     , last_modification = NEW.last_modification
+           , dataowner = NEW.dataowner
+           , provider = NEW.provider
```

Comment:
Yes, that is still there at the moment ( I know we have the fk_ now) – when we import or export data we will need a space to store it